.gitignore

```diff
@@ -162,4 +162,7 @@ cython_debug/
 #.idea/
 .python_packages
 
-local.settings.json
\ No newline at end of file
+local.settings.json
```

Comment:
@gordonhoipg We should keep local.settings.json in .gitignore. It is used only by when running functions locally and can contain secrets

Reply:
Yes, I've no intent to remove it, not sure why it included in the changes Maybe it's was accidentally changed by adding "datafiles/" in below

Reply:
you're right, misread this. must be a line ending or something